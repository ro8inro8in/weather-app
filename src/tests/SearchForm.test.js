import React from "react";
import { render } from "@testing-library/react";
import SearchForm from "../components/SearchForm";

const validProps = {
  searchText: "test search",
  setSearchText: jest.fn(),
  onSubmit: jest.fn(),
};

describe("It renders correctly", () => {
  it("renders as expected", () => {
    const { asFragment } = render(<SearchForm {...validProps} />);
    expect(asFragment).toMatchSnapshot();
  });
});
