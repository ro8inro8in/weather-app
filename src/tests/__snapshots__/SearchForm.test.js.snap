// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders correctly renders as expected 1`] = `[Function]`;
